schema.sql

```diff
@@ -12,3 +12,22 @@ CREATE TABLE animals (
 );
 
 ALTER TABLE animals ADD COLUMN species VARCHAR(50);
+
+CREATE TABLE owners (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    full_name VARCHAR(50),
+    age INT,
+    PRIMARY KEY(id)
+);
+
+CREATE TABLE species (
+    id INT GENERATED ALWAYS AS IDENTITY,
+    name VARCHAR(50),
+    PRIMARY KEY (id)
+);
+
+ALTER TABLE animals DROP COLUMN species;
+
+ALTER TABLE animals ADD COLUMN species_id INT; 
+
+ALTER TABLE animals ADD FOREIGN KEY (species_id) REFERENCES species (id);
```

Comment:
- [ ] You're also expected to add column `owner_id` which is a foreign key referencing the `owners` table